src/helpers/getHumanReadable.js

```diff
@@ -0,0 +1,2 @@
+// Returns the human-readable text based on the API key
```

Comment:
Same comment as before, but ultimately don't think you will need this.

Reply:
Removed, thank you!